### PR TITLE
chore: 🤖 upgrade to angular v15

### PR DESCRIPTION
**Pull request by chimurai**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/transloco/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Update Angular to v15
- Update nx to v15.9.2

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Comment by manianna** [comment]
Any update on this PR ? I am waiting for Angular 15 upgrade as well

**Comment by shaharkazaz** [comment]
@manianna I'm not sure how this pr affects you as a user since this is only for the internal tooling and the playground app.

File: libs/transloco-schematics/src/join/schema.ts [hunk 1]
```diff
@@ -25,5 +25,5 @@ export interface SchemaOptions extends Schema {
   /**
    * The root project name.
    */
-  project?: string;
+  project: string;
```
Comment (shaharkazaz) [inline libs/transloco-schematics/src/join/schema.ts:28]: Why did you make this change? 🤔

**Review by shaharkazaz — COMMENTED** [review]
Thanks for the PR! 🚀 
Did you happen to test all the build/tests/linked the packages?

File: libs/transloco-schematics/src/join/schema.ts (hunk 1 above)
Comment (chimurai) [reply libs/transloco-schematics/src/join/schema.ts:28]: New NX version is stricter. Getting this error when I leave `project` as optional:

image

**Comment by chimurai** [comment]
> Thanks for the PR! 🚀 Did you happen to test all the build/tests/linked the packages?

I ran these tests locally and manually checked `npm run start`:

```shell
npm run ci:test -- --skipNxCache
npm run ci:build -- --skipNxCache
npm run ci:lint -- --skipNxCache
npm run ci:e2e -- --skipNxCache
npm run lint
npm run e2e
```

Let me know if I forgot something.

File: libs/transloco-schematics/src/join/schema.ts (hunk 1 above)
Comment (chimurai) [reply libs/transloco-schematics/src/join/schema.ts:28]: `nx run transloco-schematics:build` command failed because of this inconsistency.

**Comment by krechtern** [comment]
Any news on this PR?

**Comment by chimurai** [comment]
fixed merged conflicts and upgraded to angular v16 in new PR #661

**Pull request closed without merging**